src/frontend/web_application/src/routes.jsx

```diff
@@ -42,7 +41,7 @@ export const getRouteConfig = ({ __ }) => [
       {
```

Comment:
you can remove the whole route `/discussions` I leave it before for comparison purposes